cli.js

```diff
@@ -31,6 +33,19 @@ function run() {
       process.cwd(),
     ], {stdio: 'inherit'});
     break;
+  case 'new-library':
+    if (args[1]) {
+      newLibrary(
+        process.cwd(),
+        args[1]
+      );
+    } else {
+      console.error(
+        'Usage: react-native new-library <LibraryName>'
```

Comment:
I think the latter sounds best. How about:

* `react-native new-library`
* `react-native new-component`

Both of which could take a `--platform` option, or just generate a `.ios.js` and a `.android.js` file you could delete as appropriate. I think in the spirit of keeping everything cross platform and not instacrashing, it might be better to just generate a stub every time.